examples/summary.zig

```diff
@@ -0,0 +1,48 @@
+const std = @import("std");
+const zbench = @import("zbench");
+const test_allocator = std.testing.allocator;
+
+fn fastFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..1_000) |i| {
+        std.mem.doNotOptimizeAway(i);
+        result += i * i;
+    }
+}
+fn mediumFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..20_000) |i| {
+        std.mem.doNotOptimizeAway(i);
+        result += i * i;
+    }
+}
+fn slowFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..300_000) |i| {
+        std.mem.doNotOptimizeAway(i);
+        result += i * i;
+    }
+}
+
+test "bench test summary" {
```

Comment:
It would be good to add some more test cases, like a summary for no added benchmarks, one benchmark, and the fastest being in the second and third position as well as first.

Reply:
Added more test cases 👍